src/Mvc/Mvc.Core/src/MvcOptions.cs

```diff
@@ -379,6 +380,19 @@ public int MaxModelBindingRecursionDepth
     /// <value>Defaults to <c>8192</c>.</value>
     public int MaxIAsyncEnumerableBufferLimit { get; set; } = 8192;
 
+    /// <summary>
+    /// Gets or sets a value that determines if form values are disallowed to be
+    /// parsed using <see cref="CultureInfo.InvariantCulture"/>.
```

Comment:
I'm not sure how to word this but the current culture may be invariant, meaning "disallowed" is not correct. The general sense is more like "ignore `__InvariantCulture` and do not force associated fields to be parsed using `InvariantCulture`".

Reply:
I've updated it to this:

> Gets or sets a value that determines if `<see cref="CultureInfo.CurrentCulture"/>` should always be used to parse form values.

Likewise, we could consider an alternate property name. Maybe `AlwaysUseCurrentCultureInFormModelBinding`?